src/admin-views/aggregator/status.php

```diff
@@ -0,0 +1,210 @@
+		</tr>
+	</thead>
+	<tbody>
+		<?php
+		$notes = '&nbsp;';
+		$ea_active = false;
+		if ( Tribe__Events__Aggregator::instance()->is_service_active() ) {
+			$indicator = 'good';
+			$text = __( 'Your license is active', 'the-events-calendar' );
+			$ea_active = true;
+		} else {
+			$indicator = 'bad';
+			$text = __( 'Your license is inactive', 'the-events-calendar' );
+		}
+		?>
+		<tr>
+			<td class="label"><?php esc_html_e( 'Active license', 'the-events-calendar' ); ?></td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo esc_html( $notes ); ?></td>
+		</tr>
+		<?php
+		// if EA is not active, bail out of the rest of this
+		if ( ! $ea_active ) {
+			echo '</tbody></table>';
+			return ob_get_clean();
+		}
+
+		$import_limit = Tribe__Events__Aggregator::instance()->get_daily_limit();
+		$import_available = Tribe__Events__Aggregator::instance()->get_daily_limit_available();
+		$import_count = $import_limit - $import_available;
+
+		$indicator = 'good';
+		$notes = '&nbsp;';
+
+		if ( 0 === $import_limit || $import_count >= $import_limit ) {
+			$indicator = 'bad';
+			$notes = '&nbsp;';
+			$notes = __( 'You have exceeded you daily import limit. Imports will be paused until tomorrow.', 'the-events-calendar' );
+		} elseif ( $import_count / $import_limit >= 0.8 ) {
+			$indicator = 'warning';
+			$notes = __( 'You are approaching your daily import limit, adjust you scheduled imports to avoid problems.', 'the-events-calendar' );
+		}
+
+		$text = sprintf( // import count and limit
+			_n( '%1$d import today out of %2$d available', '%1$d imports today out of %2$d available', $import_count, 'the-events-calendar' ),
+			intval( $import_count ),
+			intval( $import_limit )
+		);
+
+		?>
+		<tr>
+			<td class="label"><?php esc_html_e( 'Current usage', 'the-events-calendar' ); ?></td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo $notes; ?></td>
+		</tr>
+	</tbody>
+</table>
+
+<table class="event-aggregator-status">
+	<thead>
+		<tr class="table-heading">
+			<th colspan="4"><?php esc_html_e( 'Event Aggregator Service Status', 'the-events-calendar' ); ?></th>
+		</tr>
+	</thead>
+	<tbody>
+		<?php
+		$indicator = 'good';
+		$notes = '&nbsp;';
+
+		$ea_server = Tribe__Events__Aggregator__Service::instance()->api()->domain;
+		$up = Tribe__Events__Aggregator__Service::instance()->get( 'status/up' );
+
+		if ( ! $up || is_wp_error( $up ) ) {
+			$indicator = 'bad';
+			$notes = __( 'The Event Aggregator server is not currently responding', 'the-events-calendar' );
+		} elseif ( is_object( $up ) && is_object( $up->data ) && isset( $up->data->status ) && 400 >= $up->data->status ) {
```

Comment:
You're doing the yoda thing with the check against the HTTP status, but flipping it like that won't work since it's not a straight comparison of equality. Either needs to be `$up->data->status >= 400` (a status at or above 400) or perhaps `400 <= $up->data->status` but it can't be `400 >= $up->data->status` or it will match things like a 200 OK status.